platform/darwin/test/MGLFunctionTests.m

```diff
@@ -0,0 +1,13 @@
+#import <XCTest/XCTest.h>
+#import <Mapbox/Mapbox.h>
+
+@interface MGLFunctionTests : XCTestCase
```

Comment:
Let’s call this MGLStyleValueTests for consistency with the file that contains the implementation.